miner/payload_building_test.go

```diff
@@ -65,7 +66,7 @@ var (
 	testConfig = Config{
 		PendingFeeRecipient: testBankAddress,
 		Recommit:            time.Second,
-		GasCeil:             params.GenesisGasLimit,
+		GasCeil:             50_000_000,
```

Comment:
why this change?

Reply:
the generated transactions with the random calldata I added was overflowing the block limit without this.